test/tcs_test/tcsfresnel_molten_salt_common_data.h

```diff
@@ -0,0 +1,264 @@
+	ssc_data_set_number(data, "t_ch_out_max", 500);
+	ssc_data_set_number(data, "nodes", 2000);
+	ssc_data_set_number(data, "f_tc_cold", 2);
+	ssc_data_set_number(data, "V_tes_des", 1.85);
+	ssc_data_set_number(data, "custom_tes_p_loss", 0);
+	ssc_number_t p_k_tes_loss_coeffs[11] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+	ssc_data_set_array(data, "k_tes_loss_coeffs", p_k_tes_loss_coeffs, 11);
+	ssc_data_set_number(data, "custom_sgs_pipe_sizes", 0);
+	ssc_number_t p_sgs_diams[1] = { -1 };
+	ssc_data_set_array(data, "sgs_diams", p_sgs_diams, 1);
+	ssc_number_t p_sgs_wallthicks[1] = { -1 };
+	ssc_data_set_array(data, "sgs_wallthicks", p_sgs_wallthicks, 1);
+	ssc_number_t p_sgs_lengths[11] = { 0, 90, 100, 120, 0, 0, 0, 0, 80, 120, 80 };
+	ssc_data_set_array(data, "sgs_lengths", p_sgs_lengths, 11);
+	ssc_data_set_number(data, "DP_SGS", 0);
+	ssc_data_set_number(data, "tanks_in_parallel", 1);
+	ssc_data_set_number(data, "has_hot_tank_bypass", 0);
+	ssc_data_set_number(data, "T_tank_hot_inlet_min", 400);
+	ssc_data_set_number(data, "calc_design_pipe_vals", 0);
+	ssc_data_set_number(data, "pc_config", 0);
+	ssc_data_set_number(data, "P_ref", 111.111);
+	ssc_data_set_number(data, "eta_ref", 0.397);
+	ssc_data_set_number(data, "startup_time", 0.5);
+	ssc_data_set_number(data, "startup_frac", 0.2);
+	ssc_data_set_number(data, "q_sby_frac", 0.2);
+	ssc_data_set_number(data, "dT_cw_ref", 10);
+	ssc_data_set_number(data, "T_amb_des", 42);
+	ssc_data_set_number(data, "P_boil", 100);
+	ssc_data_set_number(data, "CT", 2);
+	ssc_data_set_number(data, "T_approach", 5);
+	ssc_data_set_number(data, "T_ITD_des", 16);
+	ssc_data_set_number(data, "P_cond_ratio", 1.0028);
+	ssc_data_set_number(data, "pb_bd_frac", 0.02);
+	ssc_data_set_number(data, "P_cond_min", 1.25);
+	ssc_data_set_number(data, "n_pl_inc", 8);
+	ssc_number_t p_F_wc[9] = { 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+	ssc_data_set_array(data, "F_wc", p_F_wc, 9);
+	ssc_data_set_number(data, "tech_type", 2);
+	ssc_data_set_number(data, "ud_T_amb_des", 43);
+	ssc_data_set_number(data, "ud_f_W_dot_cool_des", 0);
+	ssc_data_set_number(data, "ud_m_dot_water_cool_des", 0);
+	ssc_data_set_number(data, "ud_T_htf_low", 390);
+	ssc_data_set_number(data, "ud_T_htf_high", 545);
+	ssc_data_set_number(data, "ud_T_amb_low", 0);
+	ssc_data_set_number(data, "ud_T_amb_high", 55);
+	ssc_data_set_number(data, "ud_m_dot_htf_low", 0.3);
+	ssc_data_set_number(data, "ud_m_dot_htf_high", 1.2);
+	ssc_number_t p_ud_T_htf_ind_od[1] = { 0 };
+	ssc_data_set_matrix(data, "ud_T_htf_ind_od", p_ud_T_htf_ind_od, 1, 1);
+	ssc_number_t p_ud_T_amb_ind_od[1] = { 0 };
+	ssc_data_set_matrix(data, "ud_T_amb_ind_od", p_ud_T_amb_ind_od, 1, 1);
+	ssc_number_t p_ud_m_dot_htf_ind_od[1] = { 0 };
+	ssc_data_set_matrix(data, "ud_m_dot_htf_ind_od", p_ud_m_dot_htf_ind_od, 1, 1);
+	set_matrix(data, "ud_ind_od", "C:/RPP Summer/C++/SAM Hierarchy/SAM CSP Models/5 - CSP linear Fresnel molten salt/No_Financial/Fresnel_molten_salt_unit_test_default/ud_ind_od.csv", 180, 7);
```

Comment:
Fix this, and any other path issues like this in previously approved tests.

Reply:
Will do.